app/views/tenants/index.html.erb

```diff
@@ -0,0 +1,3 @@
+<% @tenants.each do |tenant| %>
+  <p><%= tenant.organization %></p>
+<% end %>
```

Comment:
Admin tenant view is to modify tenants listing, theres a user story that requires an open tenant listing.